les8/ES6/server.mjs

```diff
@@ -0,0 +1,9 @@
+// LOCAL FILE EXAMPLE
+import fib1 from './fib.mjs';
+const res1 = fib1(6);
+console.log(res1);
+
+// npm fib es6
+import fib2 from '../../node_modules/alfixedfibes6/fib.mjs';
```

Comment:
you should make it work using `import fib2 from 'alfixedfibes6`;
TIP: use webpack!
P.S. And you should not use mjs modules they are experimental!

Reply:
I fixed it )